javascript/index.js

```diff
@@ -31,35 +36,65 @@ function printMilliseconds() {
 }
 
 function printSplit() {
-  // ... your code goes here
+
 }
 
 function clearSplits() {
   // ... your code goes here
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  let stopBtn = document.getElementById('btnLeft')
+  stopBtn.classList.toggle('stop');
+  stopBtn.innerHTML = "START"
+
+  let resetBtn = document.getElementById('btnRight')
+  resetBtn.classList.toggle('split');
+  resetBtn.innerHTML = "RESET"
 }
 
+
 function setSplitBtn() {
-  // ... your code goes here
+  let splits = document.querySelector("#splits");
+  let newSplit = document.createElement("li");
+
+  newSplit.innerHTML = chronometer.splitClick();
+
+  splits.appendChild(newSplit);
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  let stopBtn = document.getElementById('btnLeft')
+  stopBtn.classList.toggle('stop');
+  stopBtn.innerHTML = "STOP"
+
+  let resetBtn = document.getElementById('btnRight')
+  resetBtn.classList.toggle('split');
+  resetBtn.innerHTML = "SPLIT"
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  chronometer.currentTime = 0;
+
+  const list = document.querySelector('#splits').parentNode
+  list.removeChild(list);
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
-  // ... your code goes here
+ if (btnLeft == document.querySelector('.stop')) {
+  setStopBtn()
```

Comment:
same as you call the startClick function below, you need to call the stopClick function here.